rust/main/lander/src/error.rs

```diff
@@ -5,7 +5,7 @@ use hyperlane_base::db::DbError;
 use crate::transaction::Transaction;
 
 #[derive(Debug, thiserror::Error)]
-pub enum SubmitterError {
+pub enum LanderError {
```

Comment:
It will be great to find if we use `SubmitterError` in string literals or comments and change it there as well.